sys/sysinfo.h not found on OpenBSD

OpenBSD is one of the target platforms for Coturn v. 4.7.0, but compilation fails with the error
```
cc  -O2 -pipe  -g  -Wall -Wno-deprecated-declarations -Wextra -Wformat-security
-Wnested-externs -Wstrict-prototypes  -Wmissing-prototypes -Wpointer-arith
-Wcast-qual  -DLIBEV_OK -DTURN_HAS_SIN_LEN -DTURN_HAS_DAEMON 
-I/usr/local/include -pthread -DTURN_NO_PROMETHEUS -DTURN_NO_SCTP  
-DINSTALL_PREFIX=/usr/local -DTURNDB=/usr/local/var/db/turndb 
-Isrc -Isrc/apps/common -Isrc/server -Isrc/client -Isrc/client++ 
-I/usr/local/include/ -I/usr/local/include/  
src/apps/oauth/oauth.c src/apps/common/apputils.c src/apps/common/ns_turn_utils.c src/apps/common/stun_buffer.c 
-o bin/turnutils_oauth -Llib -lturnclient -Llib  -pthread  -lcrypto  -lssl -L/usr/local/lib 
-levent_openssl -levent_pthreads -levent_extra -levent_core  -Wl,-rpath,/usr/local/lib

src/apps/common/apputils.c:46:10: fatal error: 'sys/sysinfo.h' file not found

#include <sys/sysinfo.h>
         ^~~~~~~~~~~~~~~
1 error generated.
*** Error 1 in /home/tmp/coturn (Makefile:127 'bin/turnutils_oauth')
```

--- a/src/apps/common/apputils.c
+++ b/src/apps/common/apputils.c
@@ -43,7 +43,6 @@
 #if defined(__unix__) || defined(unix)
 #include <pthread.h>
 #include <sys/resource.h>
-#include <sys/sysinfo.h>
 #include <sys/time.h>
 #endif
 

